docs/Releases.md

```diff
@@ -1,21 +1,24 @@
 # Releases
 
-Steps for creating a RITA release. 
+Steps for creating a RITA release.
+
+- Update the `install.sh` script so that the `_RITA_VERSION` variable reflects the desired version tag
+	- Create a branch with this change and go through the pull request process
+	- Ensure that all tests pass on this branch
+	- Note: after merging this pull request, the install script will break until you complete the rest of these steps since the installer will pull the binary file from the release page on Github, which are both yet to be created
 
-- Tag a commit on master as a release
-	- Checkout the commit
-	- Tag the commit with `git tag [version]`
```

Comment:
Since publishing a release does this anyway, I took this part out to remove one moving part.